src/Tweaks/Editor/CloseActiveDocument.cs

```diff
@@ -27,7 +27,8 @@ private static void OnKeyUp(object sender, KeyEventArgs e)
         {
             ThreadHelper.ThrowIfNotOnUIThread();
 
-            if ((e.KeyboardDevice.Modifiers & ModifierKeys.Control) == ModifierKeys.Control &&
+            if ((e.KeyboardDevice.Modifiers == ModifierKeys.Control ||
+                e.KeyboardDevice.Modifiers == (ModifierKeys.Control | ModifierKeys.Shift)) &&
```

Comment:
Wouldn't this make it close the document when either Control or Control+Alt (altgr) is pressed?

Reply:
No, it will only close if you press Ctrl or Ctrl + Shift.

Reply:
Why Ctrl + Shift?

Reply:
Of course. I just tried to bring a more proximity with the chromium behaviour. (Ctrl + Shift + W closes all pages on chromium, but that can be a future feature xD)

Reply:
Oh, I love that idea